core/orchestrator_core.go

```diff
@@ -3206,6 +4100,18 @@ func (o *TridentOrchestrator) SetVolumeState(
 }
 
 // CreateSnapshot creates a snapshot of the given volume
+// It returns an error if the snapshot cannot be created.
+// Parameters:
+//   ctx - context for logging
+//   snapshotConfig - the snapshot config
+// Returns:
+//   storage.SnapshotExternal - the snapshot external
+//   error - the error if the snapshot cannot be created, nil if the snapshot is created
+// Example:
+//   snapshotExternal, err := o.CreateSnapshot(ctx, snapshotConfig)
+//
```

Comment:
Several comments embed a double blank line after the example.  Can we prevent this?